### PR TITLE
Even more spec tweaks!

### PR DESCRIPTION
**Pull request by dmkash**
Several things in this one.  Main focus was upgrading FactoryGirl and the syntax for the factories.  Add a spec for Swayze.

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@
 require 'webmock/rspec'
 
 # Add additional requires below this line. Rails is not loaded until this point!
+require 'webmock/rspec'
```
Comment (KBalderson) [inline spec/rails_helper.rb]: duplicate of line 6 above it

File: spec/support/sunspot.rb [hunk 2]
```diff
@@ -0,0 +1 @@
+Sunspot.session = Sunspot::Rails::StubSessionProxy.new(Sunspot.session)
```
Comment (KBalderson) [inline spec/support/sunspot.rb]: oooh... I stubbed something out earlier... this looks cooler.

File: spec/rails_helper.rb (hunk 1 above)
Comment (dmkash) [inline spec/rails_helper.rb]: Good catch!

**Comment by dmkash** [comment]
Just pushed up a fix for the double-require in rails_helper

**Comment by dmkash** [comment]
@KBalderson Updating CarrierWave did fix that exception.  We can revert this change, but it sounded like the QA hit wasn't big and it would be nice to know that this works locally :smirk:

**Comment by KBalderson** [comment]
@dmkash I'll test to make sure we can still do ckeditor uploads today, then i'll merge this in

**Comment by KBalderson** [comment]
@dmkash the only issue with this is that we need to change config/initializers/carrierwave.rb#14 to use the new syntax of config.asset_host instead of config.fog_host, then it runs perfectly

**Comment by dmkash** [comment]
@KBalderson Absolutely right!  I ran into that myself and I just have neglected to update my pull request.  :frowning:

**Pull request merged**